synology_dsm/api/surveillance_station/__init__.py

```diff
@@ -0,0 +1,122 @@
+"""Synology SurveillanceStation API wrapper."""
+import urllib
+
+from .camera import SynoCamera
+from .const import MOTION_DETECTION_BY_SURVEILLANCE, MOTION_DETECTION_DISABLED
+
+
+class SynoSurveillanceStation(object):
+    """An implementation of a Synology SurveillanceStation."""
+
+    API_KEY = "SYNO.SurveillanceStation.*"
+    CAMERA_API_KEY = "SYNO.SurveillanceStation.Camera"
+    CAMERA_EVENT_API_KEY = "SYNO.SurveillanceStation.Camera.Event"
+    HOME_MODE_API_KEY = "SYNO.SurveillanceStation.HomeMode"
+    SNAPSHOT_API_KEY = "SYNO.SurveillanceStation.SnapShot"
+
+    def __init__(self, dsm):
+        """Initialize a Surveillance Station."""
+        self._dsm = dsm
+        self._cameras_by_id = {}
+
+    def update(self):
+        """Update cameras and motion settings with latest from API."""
+        response = self._dsm.get(self.CAMERA_API_KEY, "list")
```

Comment:
```suggestion
        response = self._dsm.get(self.CAMERA_API_KEY, "List")
```
Method name is case-sensitive

Reply:
All method names should be capitalized.

Reply:
OK cool, this is not the case for non SurveillanceStation APIs.
I thought is was odd.